tests/functional/frontend/FixedFunctionExpressionPluginTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
```

Comment:
Would be nice to move this to functional/acceptance/15246-fixed-em-function, maybe?